README.md

```diff
@@ -1,2 +1,77 @@
 # krux-marathon-api
-krux marathon api in python
+
+This is a command line tool to help create, delete, and keep Marathon apps in a specified state.
+It comes with a sample json config file for specifying state of an app.
+
+The Application defaults to localhost and port 8080 use the host and port flags to change the location.
+
+```./cli.py --host marathon_server.com --port 8081 --list_apps```
+
+## Create / Update Marathon Application
+Requires a valid json file (see sample).
+
+```./cli.py --config_file ../bash_nc_listen_sample.json```
+
+The application will connect to your marathon server, look to see if an application with the same name exists. If it exists already, it will update the app with the values in the json config file. If the application doesn't exist, it will create it.
+
+## Delete Marathon Application
+You can delete an app using the json file or just using the app name
+
+```./cli.py --config_file ../bash_nc_listen_sample.json --delete```
+
+```./cli.py --get_app /bash/nc-listen-test --delete```
```

Comment:
Also, `--get-app` to `--delete` feels weird. I would just make `--delete` take in a value or take `action` positional argument. Refer to above.